src/urllib3/connectionpool.py

```diff
@@ -547,7 +547,7 @@ def urlopen(  # type: ignore[override]
         pool_timeout: Optional[int] = None,
         release_conn: Optional[bool] = None,
         chunked: bool = False,
-        body_pos: Optional[Union[int, object]] = None,
+        body_pos: _TYPE_BODY_POSITION = None,
```

Comment:
urlopen should never be taking a `_FAILEDTELL` so this is more accurately `Optional[int]`.